events/2018/sponsors.md

```diff
@@ -9,78 +9,48 @@ Sponsors contribute the funds and resources that allow us to run the entire conf
-</tr>
-<tr style="height: 21px;">
-<td style="height: 21px; width: 50%;"><img src="/images/2018/web-logos/newrelic.png" alt="" style="display:block;margin:0 auto" /></td>
-<td style="height: 21px; width: 50%;"><img src="/images/2018/web-logos/recordedfuture.png" alt="" style="display:block;margin:0 auto" /></td>
-</tr>
-<tr style="height: 21px;">
-<td style="height: 21px; width: 50%;"><img src="/images/2018/web-logos/tripwire.png" alt="" style="display:block;margin:0 auto" /></td>
-<td style="height: 21px; width: 50%;"><img src="/images/2018/web-logos/twilio.png" alt="" style="display:block;margin:0 auto" /></td>
-</tr>
-<tr style="height: 64px;">
-<th style="text-align: center; height: 64px; width: 99%;" colspan="2">
-<h1><strong>Silver Sponsors</strong></h1>
-</th>
-</tr>
-<tr style="height: 21px;">
-<td style="height: 21px; width: 50%;"><img src="/images/2018/web-logos/identitytechnologies.png" alt="" style="display:block;margin:0 auto" /></td>
-<td style="height: 21px; width: 50%;"><img src="/images/2018/web-logos/isaca.png" alt="" style="display:block;margin:0 auto" /></td>
-</tr>
-<tr style="height: 21px;">
-<td style="height: 21px; width: 50%;"><img src="/images/2018/web-logos/nostarch.png" alt="" style="display:block;margin:0 auto" /></td>
-<td style="height: 21px; width: 50%;"><img src="/images/2018/web-logos/summit.png" alt="" style="display:block;margin:0 auto" /></td>
-</tr>
-<tr style="height: 64px;">
-<th style="text-align: center; height: 64px; width: 99%;" colspan="2"></th></tr>
-</tbody>
-</table>
+<div class="row">
+  <div class="columns small-12"><h1 class="center-text"><strong>Platinum Sponsors</strong></h1></div>
+</div>
+<p>&nbsp;</p>
+<div class="row">
+  <div class="columns small-6"><img src="/images/2018/web-logos/a10.png" alt="" class="center"/></div>
+  <div class="columns small-6"><img src="/images/2018/web-logos/ctrlh.png" alt="" class="center"/></div>
+  <div class="columns small-6"><img src="/images/2018/web-logos/hackerone.png" alt="" class="center"/></div>
+  <div class="columns small-6"><img src="/images/2018/web-logos/intel.png" alt="" class="center"/></div>
+  <div class="columns small-6"><img src="/images/2018/web-logos/mozilla.png" alt="" class="center"/></div>
+  <div class="columns small-6"><img src="/images/2018/web-logos/netspi.png" alt="" class="center"/></div>
+  <div class="columns small-6"><img src="/images/2018/web-logos/oracle.png" alt="" class="center"/></div>
+  <div class="columns small-6"><img src="/images/2018/web-logos/salesforce.png" alt="" class="center"/></div>
+  <div class="columns small-6"><img src="/images/2018/web-logos/securinghardware.png" alt="" class="center"/></div>
+  <div class="columns small-6"><img src="/images/2018/web-logos/symantec.png" alt="" class="center"/></div>
+  <div class="columns small-6 small-pull-6"><img src="/images/2018/web-logos/oshpark.png" alt="" class="center"/></div>
```

Comment:
This one requires the 'small-pull-6' class to move it to the left, otherwise it gets populated on the right. But also it can use 'end' See: https://foundation.zurb.com/sites/docs/grid.html#incomplete-rows